### PR TITLE
Gardener 1.54

### PR DESCRIPTION
**Pull request by majst01**

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -186,28 +186,28 @@ docker-images:
...
-        tag: v1.21.0
+        tag: v1.25.0
       virtual-apiserver:
         name: registry.k8s.io/kube-apiserver
-        tag: v1.23.17
+        tag: v1.24.15
       virtual-controller-manager:
         name: registry.k8s.io/kube-controller-manager
-        tag: v1.23.17
+        tag: v1.24.15
```
Comment (Gerrit91) [inline release.yaml:210]: For this version, we can already migrate to the community chart.

Comment (majst01) [reply release.yaml:210]: nice, but this must then be changed in our roles ? Dunno how

File: release.yaml [hunk 2]
```diff
@@ -258,5 +258,5 @@ repositories:
         ref: v0.15.4
```
Comment (majst01) [inline release.yaml:258]: @Gerrit91 
This can be removed ?

Comment (Gerrit91) [reply release.yaml:258]: No, this is our initial ETCD, it's the most important one.

Comment (majst01) [reply release.yaml:258]: I see

**Pull request merged**